aptos-move/aptos-vm/src/transaction_metadata.rs

```diff
@@ -107,6 +109,30 @@ impl TransactionMetadata {
                 }),
                 _ => None,
             },
+            payload_config: match txn.payload() {
+                TransactionPayload::Payload(TransactionPayloadInner::V1 {
+                    extra_config, ..
+                }) => match extra_config {
+                    TransactionExtraConfig::V1 {
+                        multisig_address,
+                        replay_protection_nonce,
+                    } => Some(PayloadConfig {
+                        multisig_address: *multisig_address,
+                        replay_protection_nonce: *replay_protection_nonce,
+                        scheduled_txn_auth_token: None,
+                    }),
+                    TransactionExtraConfig::V2 {
+                        multisig_address,
+                        replay_protection_nonce,
+                        permissions_table,
+                    } => Some(PayloadConfig {
+                        multisig_address: *multisig_address,
+                        replay_protection_nonce: *replay_protection_nonce,
+                        scheduled_txn_auth_token: permissions_table.get_scheduled_txn_config().ok().flatten(),
```

Comment:
this silently ignores failures from the permission tables, I think I commented somewhere we need to do validation of the table